app/routes/admin/email/components/EmailListEditor.js

```diff
@@ -2,7 +2,7 @@
 
 import React from 'react';
 import Button from 'app/components/Button';
-import { createValidator, required } from 'app/utils/validation';
+import { createValidator, required, EMAIL_REGEX } from 'app/utils/validation';
```

Comment:
There is a `isEmail` function here you can import instead of the "raw" regex

Reply:
There is, however I could not find a way to use it to validate an array of emails

Reply:
:+1: Should make some logic for validating arrayfields

Reply:
Out of scope here tho.